documentation/overview.md

```diff
@@ -0,0 +1,157 @@
+HWbench
+
+Criteo Hardware Team
+
+# Specifications
+
+- Predictive
+- Modular
+- Automated
+- Hardware driven
+
+---
+
+We have not integrated the device in prod yet
+Siena is installed on a devbox that is not representative to a final server. Fan speeds and other chassis specific metrics will be ignored to focus on the processor itself.
+
+Micro benchmarks are used as a reference point for app performance with our internal customers.
+
+Time-limited evaluation
+
+# Scripted
+
+![](img/Hwbench-intro4.png)
+
+
+
+
+- __runtime__ : absolute time in seconds per executed benchmark
+
+- __monitor__ : enable system monitoring via BMC \(redfish\)\, system \(turbostat\, sysfs\, …\)
+
+    - __Objects__:
+        - _Thermal_
+        - _Fan_
+        - _PowerConsumption_
+        - _PowerSupplies_
+        - _Frequencies_
+
+- __Support__: Generic redfish \+ OEM \(HPE / DELL\)
+
+- __engine__ : micro\-benchmark\, hwbench plugin to wrap features & command lines
+
+- __hcc__ : selects cores from the system to be eligible for benchmarking\.
+
+    - __Objects__:
+      - Logical cores \( _\<int>_ \)
+      - Physical cores \( _CORE\<int>\)_
+      - NUMA \( _NUMA\<int>_ \)
+      - Quadrant\( _QUADRANT\<int>_ \)
+      - _Simple_  \(1\, 2\, 4\, 8\, 16\, 32\.\.\.\)
+
+    Usage: interval \(0\-63\)\, list \(1\,31\)
+
+    Example: CORE0\, CORE1\, QUADRANT0\, QUADRANT0\-3
+
+- __hccs__ : selects how hosting cpu cores are associated with jobs
+
+    - __Objects__:
+      - _none _ \(all cores at once\)\
+      - _iterate _ \(each hcc value will create a new job\)
+
+- __stressor\_range__ : selects how many engine instance per job
+
+    - __Objects__:
+      - \<int>\
+      - interval \(\<x\-y>\)
+      - list \(\<x>\,\<y>\)\
+      - _auto _ \(same value as hccs\)
+
+- __\[<__  _job\_name_  _>_  __\]__ : defines one top level job
+
+- __engine\_module__ : what module from selected  _engine_
+
+- __engine\_module\_parameter__ : a list of  _engine\_module_  tests to execute
+
+# Execution (example with AMD 8434P)
+
+* System info
+* Performance
+* Monitoring
```

Comment:
These three seem out of place. The goal was to say that the json output contains all of this.